docs/sources/alerting/notifications.md

```diff
@@ -103,6 +103,10 @@ Integration Key | Integration key for PagerDuty.
 Severity | Level for dynamic notifications, default is `critical`
 Auto resolve incidents | Resolve incidents in PagerDuty once the alert goes back to ok
 
+#### Note:
+
+The tags `Class`, `Group`, and `Component` have special meaning in the [Pagerduty Common Event Format - PD-CEF](https://support.pagerduty.com/docs/pd-cef).  If an alert panel defines these tag keys they will be transposed to the root of the event sent to Pagerduty.  This means they will be avaialble within the Pagerduty UI and Filtering tools.
```

Comment:
Please remove the double spaces and fix the type in the third sentence. "available"

"Note:" should not be a heading, it should be bold and in-line with the paragraph

Reply:
Fixed, @oddlittlebird  please review